packages/orchestration/src/util/filtering.test.ts

```diff
@@ -211,4 +212,33 @@ describe('Content filter util', () => {
       );
     });
   });
+
+  describe('Llama Guard filter', () => {
+    it('builds filter config with custom config', async () => {
+      const filterConfig = buildLlamaGuardFilter({
+        child_exploitation: false,
+        hate: true,
+        violent_crimes: false,
```

Comment:
Setting properties to false can lead to confusion as skipping a property means it's false anyway

Reply:
Removed `false` properties, but the user can still set `false` as I directly use the generated type.

Reply:
No longer relevant.